lib/wpMandrill.class.php

```diff
@@ -38,29 +38,40 @@ static function on_load() {
 
             function wp_mail( $to, $subject, $message, $headers = '', $attachments = array() ) {
                 try {
-                    $sent = wpMandrill::mail( $to, $subject, $message, $headers, $attachments );
-
-                    if (
-                        is_wp_error( $sent )
-                        || !isset( $sent[0]['status'] )
-                        || in_array( $sent[0]['status'], array( 'sent', 'queued' ) )
-                        || (
-                            'rejected' === $sent[0]['status']
-                            && isset( $sent[0]['reject_reason'] )
-                            && 'hard-bounce' === $sent[0]['reject_reason']
-                        )
-                    )
-                        return;
-
-                    do_action( 'wp_mail_native', $to, $subject, $message, $headers, $attachments );
+                    $response = wpMandrill::mail( $to, $subject, $message, $headers, $attachments );
+                    self::evaluate_response( $response );
```

Comment:
Thanks, looks good! It looks like the do_action on wp_mail_native has been removed entirely. Let's put that back in there since we are now handling hard bounces. There may be other instances where the wp_mail_native function should still run.

Reply:
To be clear, this would be added after the evaluate_response function where it was previously now that we have proper error handling for certain instances.

Reply:
@MillerMedia If no exception is thrown, then Mandrill queued or sent the email, or it hard-bounced, in which case, we do not want to use `do_action( 'wp_mail_native' );`. That's why the `do_action()` is in the `catch`, and not the `try`.

Reply:
You're totally right, missed that. Looks good, I'll merge it in and we can push it out with the next release. Thanks!